CONTRIBUTING.md

```diff
@@ -0,0 +1,48 @@
+# Contributing to DoorGo
+
+## Git Workflow
+
+![GIt-Workflow-Diagram](https://user-images.githubusercontent.com/19491358/192397626-4435f01b-9b95-4245-a8a5-c85e2aa924b8.png)
+(source: http://www.davidgw.com/git-workflow-model/)
+
+## Steps
+
+1. Fork the owner's repo (upstream)
+
+   The original(owner) repo should include 2 branches: main(master), dev
+
+   - main(master) branch for production
+   - dev branch for development
+
+2. `git clone` your forked(origin) repo to local
+
+3. `git remote add upstream <repo-url>`
```

Comment:
Hey Jake, it would be better if you could briefly describe this git command.

Reply:
Sure!